tock/tock/static/js/components/notifications.js

```diff
@@ -0,0 +1,50 @@
+(function setup_notifications() {
```

Comment:
I don't think there are any set standards in this project (at least not yet), but duly noted, thanks @msecret!